packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_table_v2.dart

```diff
@@ -72,6 +74,9 @@ class _LoggingTableV2State extends State<LoggingTableV2> {
             ),
           ],
         ),
+        const SizedBox(
+          height: denseSpacing,
```

Comment:
no need for trailing comma on sized box widgets. Helps to make build methods more verbose